runtime/compiler/env/VMJ9.cpp

```diff
@@ -2407,6 +2407,18 @@ TR_J9VMBase::isHotReferenceFieldRequired()
    return TR::Compiler->om.isHotReferenceFieldRequired();
    }
 
+#if defined(J9VM_ENV_DATA64)
+/**
+ * Query if the indexable data address field is present within the indexable object header
+ *  @return true if isIndexableDualHeaderShapeEnabled is false OR if option -Xgcpolicy:balanced is specified at runtime, false otherwise
+ */
+bool
+TR_J9VMBase::isIndexableDataAddrPresent()
+   {
+   return _jitConfig->javaVM->isIndexableDataAddrPresent;
+
```

Comment:
Sentences should each end with a period.
There's an extra space before `@return` and an unnecessary blank line at 2420.